sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -17,3 +17,6 @@ mod streams;
 pub use clients::*;
 pub use parsers::*;
 pub mod models;
+
+#[cfg(test)]
+mod test_extensions;
```

Comment:
Tip: just call the mod "test". That's what we commonly do. It likely won't remain just "extension" traits forever.